documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -1119,6 +1119,8 @@ public class CarMapperImpl implements CarMapper {
 
 Additional _context_ or _state_ information can be passed through generated mapping methods to custom methods with `@Context` parameters. Such parameters are passed to other mapping methods, `@ObjectFactory` methods (see <<object-factories>>) or `@BeforeMapping` / `@AfterMapping` methods (see <<customizing-mappings-with-before-and-after>>) when applicable and can thus be used in custom code.
 
+`@Context` parameters is searched for `@ObjectFactory` methods, which are called on the provided context parameter value if applicable.
```

Comment:
shall we really do this..? Or treat the object factory methods the same for the context as for the 'uses' ? That would open up the option to handle Jaxb ObjectFactories directly as `@Context` as well.